makerdao/governance-portal-v2
# Add loading state and remove unused last vote code

### Link to Shortcut ticket:

https://app.shortcut.com/dux-makerdao/story/999/add-improved-loading-state-for-last-voted-component

### What does this PR do?

Add loading state and remove unused last vote code

### Steps for testing:

Go to a delegate detail page. See that while we fetch the last vote date, it renders a skeleton now.

### Screenshots (if relevant):

<img width="803" alt="Screen Shot 2022-04-06 at 2 57 32 PM" src="https://user-images.githubusercontent.com/5225766/161979847-9d2118df-e913-47d9-927b-ddf6d30ec4f9.png">
